src/components/CallView/Grid/Grid.vue

```diff
@@ -872,6 +872,10 @@ export default {
 	position: relative;
 	bottom: 0;
 	left: 0;
+
+	&:hover .grid-navigation {
+		background-color: rgba(0, 0, 0, 0.8) !important;
+	}
```

Comment:
Isn't it the same as `hover` and `focus` values, so there is no hover/focus effect now on those buttons?

Reply:
I left previous styles, so `hover` and `focus` works for buttons as before. But if we're moving a mouse over `<div class="wrapper">` now, buttons should also be highlighted

Reply:
There was a misunderstanding, as wrapper:hover buttons and buttons:hover have the same styles applied, we couldn't distinguish them. Should look better now with a commit below